### PR TITLE
chore: change tls path

### PR DESCRIPTION
**Pull request by mscb402**

### CODE REVIEW

File: cmd/deploy/utils.go [hunk 1]
```diff
@@ -140,10 +140,10 @@ func deployPreRun(ctx *deployContext) error {
 	if err != nil {
 		return fmt.Errorf("Failed to get default control plane: %s", err.Error())
 	}
-	if err := persistence.PrepareCertificate(cp.ID); err != nil {
+	if err := persistence.PrepareCertificate(cp.Name, cp.ID); err != nil {
 		return fmt.Errorf("Failed to prepare certificate: %s", err.Error())
 	}
-	ctx.tlsDir = persistence.TLSDir
+	ctx.tlsDir = filepath.Join(persistence.TLSDir, cp.Name)
```
Comment (nic-6443) [inline cmd/deploy/utils.go:146]: Should use `cp.id` for uniqueness.

Comment (mscb402) [reply cmd/deploy/utils.go]: done

File: cmd/deploy/kubernetes_test.go [hunk 2]
```diff
@@ -173,6 +168,7 @@ func TestKubernetesDeployCommand(t *testing.T) {
 			assert.NoError(t, err, "check if the command executed successfully")
 
 			for _, pattern := range tc.cmdPatterns {
+				fmt.Println(string(output))
```
Comment (nic-6443) [inline cmd/deploy/kubernetes_test.go:171]: Remove debug line.

**Review by nic-6443 — APPROVED** [review]

File: cmd/deploy/kubernetes_test.go (hunk 2 above)
Comment (mscb402) [reply cmd/deploy/kubernetes_test.go]: done

**Review by tokers — APPROVED** [review]

**Pull request merged**